File: dayTwo/java/level One/TaskSeven.java
public class TaskSeven{
    public static void main(String... args){

        java.util.Scanner inputCollector = new java.util.Scanner(System.in);

        System.out.print("Enter the radius: ");
        int radius = inputCollector.nextInt();

        double radiusSq = Math.pow(radius,2);

        double circumference = 2 * (Math.PI * radiusSq);
        System.out.printf("The Circumfrence of a circle  = %.02f", circumference);

    }
}

File: dayTwo/javascript/level One/taskSeven.js
const prompt = require("prompt-sync")();

let radius = Number(prompt("Enter a radius: "))

let radiusSquare = Math.pow(radius,2)
let circumference = Math.round(2 * (Math.PI * radiusSquare))

console.log(`The Circumfrence of a circle = ${circumference}`)
